app/src/main.cpp

```diff
@@ -305,7 +305,16 @@ int handleArguments( PencilApplication& app )
 int main(int argc, char* argv[])
 {
     Q_INIT_RESOURCE(core_lib);
-    PencilApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+
+    QSettings settings(PENCIL2D, PENCIL2D);
+    if (settings.value("EnableHighDpiScaling", "true").toBool())
```

Comment:
Why do you add the EnableHighDpiScaling as a setting and check it if it's always true? Is it not enough to just set the attribute always?

Reply:
Just in case we want to disable it without changing the code as it's a temporary solution.